mackup/applications/tableplus.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = TablePlus
+
+[configuration_files]
+Library/Application Support/com.tinyapp.TablePlus
```

Comment:
You also need `Library/Application Support/com.tinyapp.TablePlus/Data` in order to save the connections.